uvm/servlets/admin/app/overrides/form/field/VTypes.js

```diff
@@ -153,7 +162,7 @@ Ext.define('Ung.overrides.form.field.VTypes', {
             return true;
         }
     },
-    ipMatcherText: 'Invalid IP Address.'.t(),
+    ipMatcherText: 'Invalid IP Address. Ensure that the IP Matcher syntax is followed. The IP range needs to be in ascending order.'.t(),
```

Comment:
Let's use "Invalid IP address range, must be in ascending order."

Reply:
Sure